phase2/kubeadm/configure-vm-kubeadm-master.sh

```diff
@@ -0,0 +1,36 @@
+
+# This is not meant to run on its own, but extends phase2/kubeadm/configure-vm-kubeadm.sh
+
+KUBEADM_DIR=/etc/kubeadm
+KUBEADM_CONFIG_FILE=$KUBEADM_DIR/kubeadm.yaml
+
+#TODO: we should probably be able to configure POD_NETWORK_CIDR from `make config` in future
+# and use the configured value by passing it on to CNI's. We resort to the below hard-coding
+# since the current CNI's are not enabled to be configured with the user provided pod-network-cidr.
```

Comment:
yeah... can we create an issue that explains this?

Reply:
I will create an issue for this.